illumio/appgroups/appgroupsummary.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+"""This module provides classes related to AppGroups and AppGroupSummarys.
+
+Copyright:
+    © 2024 Illumio
+
+License:
+    Apache2, see LICENSE for more details.
+
+"""
+
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+from typing import List, Dict, Optional
+
+from illumio.util import pce_api, JsonObject
+
+
+class visibility_level_key_types(Enum):
```

Comment:
Nit: These class names should use `CapitalCase`.

Reply:
Also, please move them to `constants.py` and extend `IllumioEnumMeta` as with other enums.